inductiva/_cli/cmd_tasks/download.py

```diff
@@ -0,0 +1,56 @@
+"""Downloads a tasks by id via CLI."""
+import argparse
+import inductiva
+
+
+def download_task(args):
+    """Downloads a task by id."""
+    ids = args.id
+    filenames = args.filenames
+    output_dir = args.output_dir
+
+    if not ids:
+        print("No id(s) specified.\n"
+              "> Use `inductiva tasks download -h` for help.")
+        return 1
+
+    ids = set(ids)
+
+    if len(ids) > 1 and output_dir is not None:
+        print("Cannot specify `output_dir` for more than one download")
```

Comment:
Why not? I would actually like to just download all of these tasks into my own specified dir, with task_ids folders inside.

Reply:
That is a good point. Will add that